tests/features/asset_release/asset_release.overview.feature

```diff
@@ -21,11 +21,32 @@ Feature: Asset distribution overview on solution.
       | affiliates | Lovely Butterfly |
       | state      | validated        |
 
-    When I go to the homepage of the "Lovely Butterfly" solution
+    # Create distributions.
+    When I am logged in as a "facilitator" of the "Lovely Butterfly" solution
+    When I go to the homepage of the "Thief in the Angels" release
+    And I click "Add distribution"
+    When I fill in "Title" with "Only linux"
+    And I attach the file "test.zip" to "Add a new file"
+    And I press "Save"
+    And I go to the homepage of the "The Child of the Past" release
+    # Create a distribution with no file attached.
+    And I click "Add distribution"
+    And I fill in "Title" with "Only windows"
+    And I press "Save"
+
+    And I go to the homepage of the "Lovely Butterfly" solution
     And I click "Download"
     Then I should see the heading "Releases for Lovely Butterfly solution"
```

Comment:
I see j2 just found its codename \o/
+1 for dropping solution

(sorry I had to comment on the buttefly)